packages/mui-material/src/CssBaseline/CssBaseline.js

```diff
@@ -54,12 +55,12 @@ export const styles = (theme) => {
 /**
  * Kickstart an elegant, consistent, and simple baseline to build upon.
  */
-function CssBaseline(inProps) {
+function CssBaseline({ enableColorScheme = false, ...inProps }) {
```

Comment:
Oops, with the current implementation we would not be able to pull these from the theme's `defaultProps`. Please destructure this on line 59 after you have the props from the theme as well.

Reply:
See https://github.com/mui-org/material-ui/pull/29454/files#r742041221